doc/api/stream.markdown

```diff
@@ -181,6 +209,9 @@ readable.on('data', function(chunk) {
   console.log('got %d bytes of data', chunk.length);
 });
 ```
+Note that the `readable` event should not be used together with `data`
+because the assigning the latter switches the stream into "flowing" mode,
```

Comment:
Superfluous 'the'?  Also, is this not in direct contradiction with lines 167-168?